data-raw/qs_metabolic.R

```diff
@@ -0,0 +1,105 @@
+#' Dataset: qs_metabolic
+#' Description: Create QS test SDTM dataset for metabolic studies,
+#' containing COEQ data (Control of Eating Questionnaire)
+#' Note: the University of Leeds are the copyright holders of the CoEQ and the test data included within {admiralmetabolic}
+#' is for not-for-profit use only within `{admiralmetabolic}` and `pharmaverse`-related examples/documentation. Any persons
+#' or companies wanting to use the CoEQ should request a license to do so from the following
+#' [link](https://licensing.leeds.ac.uk/product/control-of-eating-questionnaire-coeq).
+
+# Load libraries ----
+
+library(tibble)
+library(dplyr)
+library(pharmaversesdtm)
+library(purrr)
+library(stringr)
+
+# Set seed for random data generation ----
+set.seed(3.14159265)
+
+# Read input data ----
+
+vs_metabolic <- admiralmetabolic::vs_metabolic
+
+# Set up questions, codes and categories ----
+coeq_structure <- tibble::tribble(
+  ~QSCAT, ~QSSCAT, ~QSTESTCD, ~QSTEST,
```

Comment:
I would omit QSSCAT. I don't think we would typically include such a grouping variable if it is not explicitly collected as such.

Reply:
thanks, removed